website/docs/d/projects.html.markdown

```diff
@@ -62,7 +66,15 @@ The following are valid roles:
   * `GROUP_DATA_ACCESS_READ_WRITE`
   * `GROUP_DATA_ACCESS_READ_ONLY`
   * `GROUP_CLUSTER_MANAGER`
+* `api_keys.#.api_key_id` - The unique identifier of the api key you want to associate with the project. The api key and project must share the same parent organization.
```

Comment:
```suggestion
* `api_keys.#.api_key_id` - The unique identifier of the Organization Programmatic API key assigned to the Project. 
```
Changing a bit since this is a data source.   Will add the part removed to the resource.